fe/fe-core/src/main/java/com/starrocks/connector/iceberg/io/IcebergCachingFileIO.java

```diff
@@ -120,6 +120,20 @@ public void initialize(Map<String, String> properties) {
         }
     }
 
+    @Override
+    public void close() {
+        try {
+            if (wrappedIO instanceof Closeable) {
+                ((Closeable) wrappedIO).close();
+            }
+            if (fileContentCache instanceof Closeable) {
```

Comment:
please fix the compile error

Reply:
added closable import, verified that it builds